run-build-functions.sh

```diff
@@ -616,6 +616,20 @@ install_dependencies() {
     rm -rf $GOPATH/src/$GO_IMPORT_PATH
     ln -s /opt/buildhome/repo ${GOPATH}/src/$GO_IMPORT_PATH
   fi
+
+  # Install Racket Dependencies
+  if [ -n info.rkt ]
+  then
+    raco setup
```

Comment:
The description of `raco setup` at https://docs.racket-lang.org/raco/setup.html is as follows:

> The raco setup command builds bytecode, documentation, executables, and metadata indexes for all installed collections.

Shouldn't this be something like the following?

```suggestion
    raco pkg install --deps search-auto
```

I can go ahead and push that change to your branch as well as rebasing it onto master to get it mergeable and adding cache support if you :+1: this comment. (Just don't want to force push your branch without giving you a heads up first).

Reply:
Please do! That would be awesome!